charts/polkadot-watcher-transaction/templates/alertrules.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if and .Values.prometheusRules.enabled ( ne .Values.environment "ci" ) }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+{{ toYaml .Values.prometheusRules.labels | indent 4 }}
+  name: {{ .Release.Name }}
+spec:
+  groups:
+  - name: {{ .Release.Name }}.rules
+    rules:
+    - alert: ScanStallShort
+      annotations:
+        message: 'Scanner got stuck for more than 10 minutes in {{`{{ $labels.namespace }}`}}/{{`{{ $labels.service }}`}}, please check the origin {{`{{ $labels.origin }}`}}'
+      expr: increase(polkadot_watcher_tx_scan_height[10m]) == 0
```

Comment:
I personally think scan height is pretty irrelevant for a user that receives the alert. I'd use a timestamp so the expr would be `polkadot_watcher_tx_scan_last_timestamp + 10m < now()`

Reply:
Thanks for the feedback, it made me add a clickable link to subscan... let's try to evaluate the possibilities, imo the block approach is better because:
- timestamp it's only telling when the scanner got stuck. Interesting maybe for the finance, but with no added value for us/technical team (the application logic is block based)
- if for whatever reason the scanner got stuck, the finance team should not rely on the notification channels until the stuck gets resolved
- you can click the [subscan link](https://kusama.subscan.io/block/10528217) and you have immediately the timestamp in front of you
- For us technical to solve the issue: the logic of the application is based on blocks. If we know the scan got stuck at block 10528217, maybe because the notifier(matrixbot) was down (and now is up) but the scanner's pod is in an CrashLoopBackOff state:
  - restarting the app should be more than sufficient to start from where we left
  - if for whatever reason this is not working, we can be drastic by deleting the PV of the instance, then tune the config to start a new scanner instance from block 10528216 ( one block before we got stuck )

Cons for timestamp:
- interesting for the finance, no added value for the technical team
- it would be UTC based. Meaning that the finance team needs to know that concept to make the (simple) translation. Note that also the subscan's timestamp is UTC based anyway.
- a library/dependency to add

Reply:
Ok you convinced me.